Index: src/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"Hello World 2\")\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main.py b/src/main.py
--- a/src/main.py	(revision 8afdb8603ab32d9f59b1f7f1fbe571966cb20400)
+++ b/src/main.py	(date 1637839619783)
@@ -1,1 +1,1 @@
-print("Hello World 2")
+print("Hello World 3")
